Added 8Bit music to the main game

mike4tin opened the pull request:
#114 @anand-bala let me know what you think

mike4tin committed:
8BIT README

mike4tin committed:
Added game main music

mike4tin committed:
Added comments, changed when music stops

mike4tin committed:
Merge branch 'develop' into 8bit_music

anand-bala commented on src/app/ui/ui.js:
```diff
@@ -3,12 +3,21 @@
  * @module ui/UI
  */
 /**
+ <<<<<<< 8bit_music
```
This is the beginning of a merge conflict. This has not been resolved.

anand-bala commented on src/app/ui/ui.js:
```diff
@@ -3,12 +3,21 @@
+ * @param {string} text
+ * @param {*} key object associated with button
+ * @param {*} func
+ =======
```
This is the divider between incoming (your) changes and current file.

anand-bala commented on src/app/ui/ui.js:
```diff
@@ -3,12 +3,21 @@
  * @param {string} text 
  * @param {string} key - object associated with button
  * @param {function} func - on click callback function
+ >>>>>>> develop
```
This is the end of the previously marked conflict. This one is simple, just remove the first section (from the `8bit_music` line to the `===`), and remove the marking lines

anand-bala reviewed (changes requested):
You seem to be committing a new `package-lock.json`. It would be preferable if you replaced this with what was originally in the repo

mike4tin committed:
Fixing requested changes

anand-bala commented:
@mike4tin  This works great, but I am going to hold off merging it for a bit. Thanks a tonne.

mike4tin committed:
Added attribution for music

Adds attribution to main menu screen as required by the current music's license requirements. Also adds a music information file with all of the same details

mike4tin committed:
Changed the name of the song

I had the wrong name to the song, updated in both notices

anand-bala commented:
@mike4tin can you fix the merge conflicts?

mike4tin commented:
Not sure if I can. I can't open it in terminal like I would normally be able to if it was my repo

mike4tin committed:
Merge pull request #1 from PendragonGame/develop

keeping up to date

mike4tin committed:
Trying to fix conflicts
